### PR TITLE
TVB-2905: Upload missing files when importing simulation from ZIP

### PR DESCRIPTION
**Pull request by VinczeRobert**

### CODE REVIEW

File: framework_tvb/tvb/core/entities/file/files_update_manager.py [hunk 1]
```diff
@@ -242,16 +242,24 @@ def get_all_h5_paths():
...
-    def _get_create_date_for_sorting(h5_file):
-        create_date_str = str(H5File.get_metadata_param(h5_file, 'Create_date'), 'utf-8')
-        create_date = string2date(create_date_str, date_format='datetime:%Y-%m-%d %H:%M:%S.%f')
+    def _get_create_date_for_sorting(h5_file, version_5):
+        if version_5:
+            create_date_str = H5File.get_metadata_param(h5_file, 'create_date')
+            create_date = string2date(create_date_str, date_format='%Y-%m-%d,%H-%M-%S.%f')
+        else:
+            create_date_str = str(H5File.get_metadata_param(h5_file, 'Create_date'), 'utf-8')
+            create_date = string2date(create_date_str, date_format='datetime:%Y-%m-%d %H:%M:%S.%f')
```
Comment (popaula937) [inline framework_tvb/tvb/core/entities/file/files_update_manager.py:263]: are these changes still necessary?

File: framework_tvb/tvb/tests/framework/core/neocom_test.py [hunk 2]
```diff
@@ -132,14 +132,14 @@ def test_gather_view_model_references(connectivity_index_factory, operation_fact
     storage_path = FilesHelper().get_project_folder(op.project, str(op.id))
     h5.store_view_model(sim_view_model, storage_path)
 
-    only_vm_references = h5.gather_view_model_references(sim_view_model.gid, storage_path, True)
-    all_references = h5.gather_view_model_references(sim_view_model.gid, storage_path)
+    only_vm_references = h5.gather_view_model_and_datatype_references(sim_view_model.gid, storage_path, True)[0]
```
Comment (popaula937) [inline framework_tvb/tvb/tests/framework/core/neocom_test.py:135]: Please always use:
`only_vm_references, _ = h5.gather_view_model_and_datatype_references(...)`
instead of:
`only_vm_references = h5.gather_view_model_and_datatype_references(...)[0]`

File: framework_tvb/tvb/core/entities/file/files_update_manager.py (hunk 1 above)
Comment (VinczeRobert) [reply framework_tvb/tvb/core/entities/file/files_update_manager.py]: I now undid the changes in files_update_manager.py.

File: framework_tvb/tvb/tests/framework/core/neocom_test.py (hunk 2 above)
Comment (VinczeRobert) [reply framework_tvb/tvb/tests/framework/core/neocom_test.py]: Done

**Review by popaula937 — APPROVED** [review]
